src/main/java/com/epam/ta/reportportal/dao/UserRepositoryCustomImpl.java

```diff
@@ -42,6 +33,7 @@
 @Repository
 public class UserRepositoryCustomImpl implements UserRepositoryCustom {
 
+	@Qualifier("localDataStore")
```

Comment:
data store should be configurable. and it has been already implemented. you can just remove it